tests/integration/routes/test_view_submission.py

```diff
@@ -88,16 +84,9 @@ def test_try_view_submission_when_not_available(self):
         # Submit answers
         self.post(action=None)
 
-        # check we're on the thank you page and view submission link is not available
```

Comment:
This test method name no longer matches what is actually being tested. It should end on an assertion. I think the only thing that needed to be removed was the "self.assertNotInBody"?